### DISCUSSION TITLE
Using the same mdx file for various stories without it also being listed as unattached?

### DISCUSSION THREAD

**Discussion by IgnacioBecerra** [storybookjs/storybook#25919] [Help]
### Summary

We have a few stories under the same directory, and all should be pointing to it's corresponding component doc file, in this case `data-table.mdx`. Each story folder should have its own "Overview" doc file, which still points to the same mdx file. As such, we can't use the `Meta` in the mdx file as it would only be attached to a single story. By refraining from using it, it works, but then we have a stray unattached doc file in our story list (notice the top). 

<img width="227" alt="Screenshot 2024-02-05 at 4 52 47 PM" src="https://github.com/storybookjs/storybook/assets/24970122/5f40b678-03b7-4b36-a9ce-a0b2dd014102">

I know it is possible to remove the stray file somehow, but I'm having a difficult time figuring out how to do so. Any possible suggestions? Thank you!

### Additional information

Storybook version: `v7.6.4`

`main.js`
``` 
// rest of config JSON
  docs: {
    autodocs: true,
    defaultName: 'Overview',
  },
// etc
```

Every `data-table-*.stories.ts` file:

```
const meta = {
  title: 'Components/DataTable/Basic', // changes per story
  parameters: {
    docs: {
      page: storyDocs,
    },
  },
};
```

**Comment by asynchroza** [comment]
Did you come by a solution?

**Comment by smlimasu** [comment]
You can disable the doc in the sidebar by setting <Meta isTemplate> or !dev in the tags. 

Alternatively, take a look at autodocs, where you set the page: autodocstemplate in preview or at story meta and this autodocs file can be any template you want, either MDX or a react component 

With isTemplate the doc has meta, but takes the story files context as meta if you need to display f.ex. each argstable differently for each component story file
